### ISSUE TITLE
Proxy usage with SSL connection

### ISSUE THREAD

**Issue by temka1234** [TooTallNate/Java-WebSocket#175]
Hi, Is there a way to use **proxy** with **wss**, not **ws**, in your WebSocket implementation?
Thanks.

**Comment by shuckc** [comment]
I've just tested the combination of wss:// + proxy in v1.3.0 and it does not work together. I can see in Wireshark that the CONNECT header is not being sent in plaintext. It needs to be exchanged for a "200 Connection established" header with the proxy before the SSL initialisation begins.

I can see there are some changes in master though, let me try from a checkout.

**Comment by shuckc** [comment]
Per my comments on #205 proxy + wss works using code from master works for us

**Comment by marci4** [comment]
Closing the issue due to @shuckc comment.

Greetings
marci4

**Comment by daluu** [comment]
RE: @shuckc's comment, anyone know if it works for Android or just in (desktop) Java? For Java, there seems to be a [platform limitation](https://bugs.java.com/bugdatabase/view_bug.do?bug_id=6370908) that was fixed a while back (in Java 8?). But not aware of anything similar for Android, there's no mention of the issue or whether there is or isn't HTTP proxy support for sockets in Android. Just asking as it seems we've encountered the mentioned Java limitation error using this library with HTTP proxy specified on Android.
